engine/src/main/scala/cromwell/webservice/WorkflowJsonSupport.scala

```diff
@@ -43,7 +43,7 @@ object WorkflowJsonSupport extends DefaultJsonProtocol {
     }
   }
 
-  implicit val workflowQueryResult = jsonFormat5(WorkflowQueryResult)
+  implicit val workflowQueryResult = jsonFormat6(WorkflowQueryResult)
```

Comment:
I know you have checkboxes for updating the README and CHANGELOG, but we may want to make sure our big customers are aware of the change in this format.